modin/engines/dask/pandas_on_dask/frame/partition_manager.py

```diff
@@ -20,29 +20,34 @@
 )
 from .partition import PandasOnDaskFramePartition
 from modin.error_message import ErrorMessage
+import pandas
 
 from distributed.client import _get_global_client
 import cloudpickle as pkl
 
 
-def deploy_func(df, other, apply_func, call_queue_df=None, call_queue_other=None):
+def deploy_func(df, apply_func, call_queue_df=None, call_queues_other=None, *others):
```

Comment:
What is the purpose of this change?

Reply:
`deploy_func` is a remote function, when it obtains future it will be materialized inside that function, so since we need to pass a variable number of `others` futures,  we're packing it in a `*others`. (if we pass a list of futures inside the function it will be still a list of unmaterialized futures)